apps/artist/client/views/header.coffee

```diff
@@ -31,6 +31,7 @@ module.exports = class ArtistHeaderView extends Backbone.View
         following: view.following
         modelName: 'artist'
         model: view.model
+        authOnClick: !SHOW_ARTIST_CTA
```

Comment:
So here if we aren't showing the artist cta we want to trigger auth on the click?